spyderlib/widgets/explorer.py

```diff
@@ -1000,64 +1000,82 @@ class ExplorerWidget(QWidget):
     def __init__(self, parent=None, name_filters=['*.py', '*.pyw'],
                  show_all=False, show_cd_only=None, show_icontext=True):
         QWidget.__init__(self, parent)
-        
+
+        # Widgets        
         self.treewidget = ExplorerTreeWidget(self, show_cd_only=show_cd_only)
-        self.treewidget.setup(name_filters=name_filters, show_all=show_all)
-        self.treewidget.chdir(getcwd())
-        
+        self.button_previous = QToolButton(self)
+        self.button_next = QToolButton(self)
+        self.button_parent = QToolButton(self)
+        self.button_menu = QToolButton(self)

```

Comment:
I think we can safely remove the `self` from all these `QToolButton`'s, right? I mean, it seems they aren't referenced outiside `__init__`